src/lib/Importer/SchemaImporter.php

```diff
@@ -176,6 +190,60 @@ private function ensureNoExtraKeys(array $tableConfiguration, string $location,
             ));
         }
     }
+
+    /**
+     * @param array<string, mixed> $indexConfig
```

Comment:
On one hand, yes - the expected argument data is well known. It can be normalized using the preceeding method. If the initial value was a string, it will be converted into
```
array{
    name: string,
    unique?: bool,
    options?: array<mixed>,
}
```
My initial thought was that since this data comes from user in general, then we should perform all necessary type checks. But you are probably right. And we can rely on PHP to perform type checks here.

Just need to add a `try...catch` for Throwable so that we do actually provide the location for the error, I guess.

Reply:
Fixed in 605d024e.